src/sentry/models/organizationmember.py

```diff
@@ -89,6 +97,18 @@ class OrganizationMember(Model):
     teams = models.ManyToManyField(
         "sentry.Team", blank=True, through="sentry.OrganizationMemberTeam"
     )
+    inviter = FlexibleForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True, related_name="sentry_inviter_set"
+    )
+    invite_status = models.PositiveSmallIntegerField(
+        choices=(
+            (InviteStatus.APPROVED.value, _("Approved")),
+            (InviteStatus.REQUESTED_TO_BE_INVITED.value, _("Org member requested to invite user")),
```

Comment:
Do these strings show to the end user? Should we use 'Organization' instead of 'Org'?